core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -120,6 +120,76 @@ paths:
             - profile
             - https://www.googleapis.com/auth/cloud-billing
       x-codegen-request-body-name: createProjectRequest
+  /api/billing/v2/{projectId}/billingAccount:
```

Comment:
This swagger change is ripped right from `v2_workspaces`. Only change is that I return `OK` with no body instead of `OK` with a `RawlsBillingProjectResponse`. I left another comment on that in the route layer